Horizon/Horizon/Sources/Features/Learn/Scores/Data/HAssignmentGroup.swift

```diff
@@ -19,47 +19,39 @@
 import Core
 import Foundation
 
-struct HAssignmentGroup: Identifiable {
-    let id: String
-    let name: String
-    let groupWeight: Double?
-    var groupWeightString: String? {
-        if let groupWeight {
-            return GradeFormatter.numberFormatter.string(
-                from: NSNumber(value: groupWeight)
-            )
-        } else {
-            return nil
-        }
-    }
-
-    let assignments: [HAssignment]
+ struct HAssignmentGroup: Identifiable {
```

Comment:
Let's rename this to `ScoresAssignmentGroup`. We don't need need the `H` prefix as there's no such an object in Core.